design/components/cards.tsx

```diff
@@ -0,0 +1,89 @@
+import { useLink } from '@react-aria/link';
+import React, { useRef } from 'react';
+
+import { RichText } from './rich-text';
+import { styled } from './stitches.config';
+import { Tag } from './tag';
+import { Title } from './title';
+
+const StyledStories = styled('a', {
+  img: { height: 324, width: 406 },
+  position: 'relative',
+  outline: 'none',
+});
+
+const StyledCases = styled('a', {
+  img: { height: 324, width: 406 },
+  position: 'relative',
+  outline: 'none',
+});
+const ContentContainer = styled('div', {
+  display: 'flex',
+  position: 'absolute',
+  top: -40,
+  left: 0,
+  flexFlow: 'column nowrap',
+  width: 376,
+  backgroundColor: '#ffffff',
+  main: {
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    alignContent: 'space-between',
+    margin: '$medium',
+  },
+});
```

Comment:
Can you pad your export lines with empty lines.

Reply:
i didn't get that. what i suppose to do here ?